src/vdaf/prio3.rs

```diff
@@ -266,10 +259,10 @@ pub fn prio3_start<V: Value>(
     deriver.update(nonce);
     let query_rand_seed = deriver.finish();
 
-    let input_share_data: Vec<V::Field> = Vec::try_from(msg.input_share)?;
+    let input_share_data: Vec<V::Field> = msg.input_share.into_vec(param.input_len())?;
     let input_share = V::new_share(input_share_data, param, verify_param.num_shares as usize)?;
 
-    let proof_share_data: Vec<V::Field> = Vec::try_from(msg.proof_share)?;
+    let proof_share_data: Vec<V::Field> = msg.proof_share.into_vec(param.proof_len())?;
```

Comment:
Are the type annotations on `input_share_data` and `proof_share_data` needed anymore since we don't have to give the compiler a hint on which `try_from` to use?

Reply:
Nope. Removed.